ProtonVPN/ServerInfo.xib

```diff
@@ -10,7 +10,7 @@
     <objects>
         <customObject id="-2" userLabel="File's Owner" customClass="ServerInfoViewController" customModule="ProtonVPN" customModuleProvider="target">
             <connections>
-                <outlet property="bottonConstraint" destination="Yya-Tn-hs2" id="lJK-PE-4gu"/>
+                <outlet property="buttonConstraint" destination="Yya-Tn-hs2" id="lJK-PE-4gu"/>
```

Comment:
It's always hard to figure out if people want `button` or `bottom` when they write `botton`. I've made a guess, but I have zero confidence in my guess.